readme.md

```diff
@@ -54,13 +54,10 @@ Compilation
 Follow the instructions at the [megasource][megasource] repository page.
 
 ### *nix
-Run `platform/unix/automagic` from the repository root, then run ./configure and make.
+First pick a directory to store generated files to, say `build/` and then:
 
-	$ platform/unix/automagic
-	$ ./configure
-	$ make
-
-When using a source release, automagic has already been run, and the first step can be skipped.
+	$ cmake -B build -S. --install-prefix $PWD/prefix # this will create the directory `build/`
+	$ cmake --build build --target install # this will put the files in `prefix/`.
```

Comment:
`build` and `prefix` folders will popup as untracked files in git repository. Is it ok ?

Reply:
`cmake -S.` also implies that we run these commands from the repository root. Probably we should explicitly mention it

Reply:
As I see in CMakeLists.txt, you can't run `cmake -B. -S.`, but it should be ok for `cmake -Bbuild -S.`